cl/recap/tests/test_recap_email.py

```diff
@@ -3108,3 +3122,958 @@ async def test_nda_document_not_available_get_from_confirmation_page(
+        self.assertEqual(
+            await all_pqs_created.acount(),
+            13,
+            msg="Wrong number of ProcessingQueues.",
+        )
+
+        # 7 PQs related to the main document:
+        # One for storing the main PDF from the notification.
+        # Two for storing the attachment page from the notification.
+        # Two for replicating the main PDF to two subdockets.
+        # Two for replicating the attachment page to two subdockets.
+        main_pqs_created = ProcessingQueue.objects.filter(
+            pacer_doc_id="85001321035"
+        )
+        self.assertEqual(
+            await main_pqs_created.acount(),
+            7,
+            msg="Wrong number of ProcessingQueues.",
+        )
+
+        # 3 PQs related to PDF attachments:
+        # One for storing the att PDF from the notification.
+        # Two for replicating the att PDF to two subdockets.
+        att_pqs_created = ProcessingQueue.objects.filter(
+            pacer_doc_id="85001321036"
+        )
+        self.assertEqual(
+            await att_pqs_created.acount(),
+            3,
+            msg="Wrong number of ProcessingQueues.",
+        )
+        async for rd in recap_documents:
+            with self.subTest(rd=rd):
+                # Every RECAPDocument should have a file stored at this point.
+                self.assertTrue(rd.filepath_local)
+                self.assertTrue(rd.is_available)
+                if not rd.attachment_number:
+                    # Check that every main RECAPDocument has the main pacer_doc_id
+                    self.assertEqual(rd.pacer_doc_id, "85001321035")
+                if rd.attachment_number == 1:
+                    # Check that every attachment RECAPDocument has the attachment
+                    # pacer_doc_id
+                    self.assertEqual(rd.pacer_doc_id, "85001321036")
+
+        # 2 DocketAlert email for the recap.email user should go out
+        self.assertEqual(len(mail.outbox), 2)
+
+    @mock.patch(
+        "cl.recap.tasks.get_pacer_cookie_from_cache",
+        side_effect=lambda x: True,
+    )
+    @mock.patch(
+        "cl.recap.tasks.download_pdf_by_magic_number",
+        side_effect=lambda z, x, c, v, b, d, e: (
+            MockResponse(200, b"Hello World"),
+            "OK",
+        ),
+    )
+    @mock.patch(
+        "cl.recap.tasks.requests.get",
+        side_effect=lambda *args, **kwargs: MockResponse(200, b"Att content."),
+    )
+    async def test_avoid_triggering_replication_for_minute_entries(
+        self,
+        mock_att_request,
+        mock_download_pdf,
+        mock_cookie,
+        mock_docket_entry_sealed,
+        mock_pacer_court_accessible,
+        mock_cookies,
+        mock_bucket_open,
+        mock_enqueue_alert,
+    ):
+        """Confirm the main PDF and the related attachment data and attachment
+        PDF are properly replicated from a multi-nef notification to subdockets.
+        """
```

Comment:
This docstring is a duplicate of the one used in the previous test. It doesn't accurately describe the purpose of this specific test case.

Reply:
Sure, I've updated these docstrings!